content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -910,7 +911,7 @@ kubectl delete pod dapi-test-pod --now
 
 # You might already have removed the next set
 kubectl delete configmaps/special-config configmaps/env-config
-kubectl delete configmap -l 'game-config in (config-4,config-5)’
+kubectl delete configmap -l 'game-config in (config-4,config-5)'
```

Comment:
Similar changes also require to do in line number 529. Please also update that.

Reply:
@1000Delta,
Please verify throughout the page if there is anything else to improve and update all necessary changes.

Reply:
Done